build.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+set -e
+set +x
+
+trap "cd $(pwd -P)" EXIT
+cd "$(dirname "$0")"
+
+if [[ ($1 == '--help') || ($1 == '-h') ]]; then
+  echo "usage: build.sh <command>"
+  echo "commands:"
+  echo "  --init            - download .NET deps, download driver, update assets"
+  echo "  --roll <version>  - roll the .NET language binding to a specific driver version"
+  echo "  --download-driver - download the driver"
+  echo "  --help            - show this help"
+  echo
+  exit 0
+fi
+
+if [[ $# == 0 ]]; then
+  echo "missing command!"
+  echo "try './build.sh --help' for more information"
+  exit 1
+fi
+
+upstream_repo_path="../playwright"
+if [[ -n "$PW_SRC_DIR" ]]; then
+  upstream_repo_path="$PW_SRC_DIR"
+fi
+echo "Upstream repo path: ${upstream_repo_path}"
+
+function download_dotnet_tool_deps() {
+  echo "Downloading .NET tool dependencies"
+  dotnet tool install --global dotnet-format || true
+  echo "done"
+}
+
+function download_driver() {
+  echo "downloading driver..."
+  dotnet run --project ./src/tools/Playwright.Tooling/Playwright.Tooling.csproj -- download-drivers --basepath .
+  echo "done"
+}
+
+function update_assets() {
+  echo "updating assets..."
+  dotnet_assets_path="./src/Playwright.Tests.TestServer/wwwroot"
+  rm -rf "$dotnet_assets_path"
+  cp -r "${upstream_repo_path}/tests/assets" "${dotnet_assets_path}"
+  echo "done"
+}
+
+function roll_driver() {
+  new_driver_version="$1"
+  upstream_package_version=$(node -e "console.log(require('${upstream_repo_path}/package.json').version)")
+  echo "Rolling .NET driver to driver ${new_driver_version} and upstream version ${upstream_package_version}..."
+
+  xml_file_path="./src/Common/Version.props"
+  xml_file_contents=$(cat "${xml_file_path}")
+  xml_file_contents=$(echo "${xml_file_contents}" | sed "s/<DriverVersion>.*<\/DriverVersion>/<DriverVersion>${new_driver_version}<\/DriverVersion>/")
```

Comment:
nit: you can use any symbol for sed scripts to separate parts of expression.

For example:
```bash
sed "s/foo/bar" # default
sed "s|foo|bar" # variations
sed "s#foo#bar"
sed "s foo bar" # or even space would do!
```